x-pack/dockerlogbeat/pipelinemanager/pipelineManager.go

```diff
@@ -89,20 +103,82 @@ func (pm *PipelineManager) CreateClientWithConfig(containerConfig ContainerOutpu
 
 	reader, err := pipereader.NewReaderFromPath(file)
 	if err != nil {
-		return nil, errors.Wrap(err, "")
+		return nil, errors.Wrap(err, "error creating reader for docker log stream")
+	}
+
+	// Why is this empty? What should be here? Who knows!
+	if info.LogPath == "" {
+		info.LogPath = filepath.Join("/var/log/docker", info.ContainerID)
```

Comment:
this requires users to always mount /var/log/docker into our plugin. Do we have some means to detect that this is not the case and either error or introduce our own cleanup logic?

Reply:
This is a default, I believe. There's no special mounting going on plugin side or container side, unless I'm missing something.

Reply:
Nice, I didn't expect the hosts /var/log/docker to be mounted into the plugin container